backend/src/routes/user.route.ts

```diff
@@ -7,6 +8,7 @@ const router = express.Router();
 
 router
   .get('/', userController.getAll)
+  .post('/uploadImage', uploadImage)
```

Comment:
Could we use previous convention? 

`Controller.method`
in this case could be `cloudinaryController.upload`